Checkout/Checkout/Test/Model/CardSchemeTests.swift

```diff
@@ -1,83 +0,0 @@
-//
-//  Card.SchemeTests.swift
-//
-//
-//  Created by Harry Brown on 21/10/2021.
-//
-
-import XCTest
-@testable import Checkout
-
-class CardSchemeTests: XCTestCase {
```

Comment:
I'm not sure you were supposed to remove existing unit tests?

Reply:
Nice catch